ui/components/imageWorkspace/src/components/SaveShape/SaveShapeForm.svelte

```diff
@@ -82,10 +82,16 @@
 
       return [...oldObjects, ...(newObject ? [newObject] : [])];
     });
-    newShape.set({ status: "none" });
+    newShape.set({ status: "none", shouldReset: true });
     canSave.set(true);
     currentTab = "objects";
   };
+
+  async function handleKeyDown(event: KeyboardEvent) {
```

Comment:
question : why did you add this function here and not in canvas?

Reply:
because newShape is a Writable<Shape> here in SaveShapeForm, but a Shape in Canvas2D, so shpae doesn't have the set function needed here to close the form